home/ivp-planning/user-interface/the-floor-plan.md

```diff
@@ -19,4 +19,4 @@ The [floor plan](the-floor-plan.md) - the virtual room or factory hall in which
 
 ## PDFs as floor plan:
 
-It is possible to use a PDF as floor plan by [importing it](../getting-started/importing-pdfs.md). If the project already contains a PDF, it is possible to toggle its [visibility ](../getting-started/pdf-visibility.md)or [delete ](../getting-started/delete-pdfs.md)it. It can be replaced by [importing ](../getting-started/importing-pdfs.md)another PDF.
\ No newline at end of file
+It is possible to use a PDF as floor plan by [importing it](../getting-started/importing-pdfs.md). If the project already contains a PDF, it is possible to toggle its [visibility](../getting-started/pdf-visibility.md) or [delete](../getting-started/delete-pdfs.md) it via the [hierarchy panel](../user-interface/hierarchy-panel.md).
```

Comment:
This is only the case when loading an old plan where a Pdf was saved this way. We should make clear that this is a legacy option that is only relevant when loading old plan files.